src/user-guide/create-plot.md

```diff
@@ -1 +1,56 @@
+
 # Creating a plot
+
+One of RABIT's premier features is the creation of Bayesian inference plots.
+
+
+
+
+## 1. Navigate to the upload page
+
+Navigate via the sidebar or go directly to the '/upload' URL. On the upload page, users will be able to create a plot by filling out the mandatory fields.
+
+![App Screenshot](../images/create-plot/navigation.png)
```

Comment:
Rename link to `../images/create-plot/navigation.PNG` or image filename to `navigation.png`

(note: you need to rename all instances)